webapp/portlet/src/main/webapp/vue-apps/activity-reactions/components/ActivityReactionsDrawer.vue

```diff
@@ -5,68 +5,33 @@
     right
     fixed>
     <template slot="title">
-      <div class="activityReactionsTitle">
-        <v-tabs v-model="selectedTab">
-          <v-tab class="allLikersAndKudos text-color pe-3 ps-0" href="#tab-1">{{ $t('UIActivity.label.Show_All_Likers') }} {{ reactionsNumber }}</v-tab>
-          <v-tab class="allLikers pe-3 ps-0" href="#tab-2"><i class="uiIconThumbUp"></i> <span class="primary--text">{{ likersNumber }}</span></v-tab>
+      {{ $t('UIActivity.label.Reactions_Number') }}
+    </template>
+    <template v-if="drawerOpened" slot="content">
+      <div>
+        <v-tabs
+          fixed-tabs
+          slider-size="4"
+          v-model="selectedTab">
           <v-tab
             v-for="(tab, i) in enabledReactionsTabsExtensions"
             :key="i"
-            :href="`#tab-${tab.order}`"
-            :class="`all${tab.class}`"
-            class="pe-3 ps-0">
-            <i :class="tab.icon"></i>
-            <span :class="`${tab.class}NumberLabel`">{{ tab.kudosNumber }}</span>
+            :href="`#tab-${tab.componentOptions.order}`"
+            class="text-capitalize">
+            <span>{{ tab.componentOptions.reactionType }} ({{ tab.componentOptions.numberOfReactions }})</span>
```

Comment:
`reactionType` should be internationalised.
```suggestion
            <span>{{ tab.componentOptions.reactionType }} ({{ tab.componentOptions.numberOfReactions || '' }})</span>
```